content/sensu-go/5.0/reference/agent.md

```diff
@@ -654,7 +654,7 @@ keepalive-timeout: 300{{< /highlight >}}
 
 | namespace |      |
 ---------------|------
-description    | Agent namespace
+description    | Agent namespace _NOTE: Agents can only have one namespace._
```

Comment:
```suggestion
description    | Agent namespace _NOTE: Agents can only belong to a single namespace._
```

Reply:
Perhaps we should go a little further to say `Agents are represented in the backend a class of entity. Entities can only belong to a single namespace.` with a link to the table of namespaced resources